fastlane/Screengrabfile

```diff
@@ -1,12 +1,12 @@
 
-#locales ['en','hi','in','jv','su','th','tl','zh-TW','vi']
+#locales ['en','hi','in','jv','su','th','tl','zh-TW','vi','ta','kn','ml']
 locales ['vi','zh-TW']
 clear_previous_screenshots true
 exit_on_test_failure false
 test_instrumentation_runner  'org.mozilla.focus.test.runner.ScreenshotTestRunner'
-app_package_name 'org.mozilla.rocket.debug.firebase'
+app_package_name 'org.mozilla.rocket.debug.pliu'
```

Comment:
maybe add a comment or update the wiki so the future maintainer could know why it's needed.